pagesnotfound.php

```diff
@@ -103,6 +103,8 @@ private function getPages()
 
     public function hookDisplayAdminStatsModules()
     {
+	$this->context->controller->addCSS($this->_path . 'views/css/stacking_responsive.css');
```

Comment:
```suggestion
        $this->context->controller->addCSS($this->_path . 'views/css/stacking_responsive.css');
```
Invalid indentation - looks like 1 tab. Should be 8 spaces leading this line.

Reply:
It is a tab - inherited from old style codes, line 107 is really empty now.